docs/user/reference/juju-cli/list-of-juju-cli-commands/bootstrap.md

```diff
@@ -201,6 +201,9 @@ Bootstrap configuration keys:
         Controls the kubernetes service type for Juju controllers, see
         https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#ServiceSpec
         valid values are one of cluster, loadbalancer, external
+    ssh-server-host-key:
+      type: string
+      description: Sets the bootstrapped controllers SSH server host key
```

Comment:
Missing apostrophe.

Can we also mark this as optional so the user knows they don't need to provide a value.

Reply:
Done